src/content/docs/nrql/nrql-syntax-clauses-functions.mdx

```diff
@@ -737,6 +737,43 @@ As noted in our [basic NRQL syntax doc](/docs/query-your-data/nrql-new-relic-que
     </Callout>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="#sel-prediction"
+    title={<><InlineCode>PREDICT</InlineCode> clause</>}
+  >
+   <Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+    ```sql
+    FROM CrossClusterQuery SELECT count(*) WHERE status NOT IN ('successful', 'rejected') TIMESERIES PREDICT
+    ...
+    ```
+
+    With the `PREDICT` clause, you can incorporate predictions regarding future data trends into line and area charts based on historical data. This clause:
+
+    * Applies only to queries with a `TIMESERIES` clause.
+    * Uses the `TIMESERIES <time period>` as the interval for predicted data points.
+    * Sets the prediction range to 20% of the query window by default.
+    * Processes historical data from the current query window and the two preceding ones to generate predictions by default.
+    * Does not support [metric timeslice data](/docs/data-analysis/metrics/analyze-your-metrics/data-collection-metric-timeslice-event-data#timeslice-data) while in Public Preview.
+
+    <Callout variant="important">
+      The `PREDICT` clause is not supported when used with the [`COMPARE WITH`](/docs/nrql/nrql-syntax-clauses-functions/#sel-compare) clause, [nested aggregations](/docs/nrql/using-nrql/nested-aggregation-make-ordered-computations-single-query) inside [subqueries](/docs/nrql/using-nrql/subqueries-in-nrql/).
+    </Callout>
+    You can additionally append `PREDICT <model>`, `BY <time range>`, and `USING <time range>` to refine the prediction trend to your specific needs. 
+    To indicate the time range and historical data span, use `integer units`.
+
+            ```sql
+        FROM CrossClusterQuery SELECT count(*) WHERE status NOT IN ('successful', 'rejected') TIMESERIES PREDICT holtwinters(beta: 0.5) BY 3 hours USING 1 day
```

Comment:
Same as above, CrossClusterQuery is an internal monitoring event that's not accessible to customers. Let's change this to "FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES PREDICT holtwinters(beta: 0.5) BY 3 hours USING 1 day"